apm-hub-helm/templates/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
```

Comment:
This will cause issues when deploying multiple charts into the same namespace, it also isn't linked to the resources - i.e helm could deploy the resources into a different namespace than this.

We can remove this entirely - this chart will depend on an existing namespace,

Reply:
Ya... I missed removing this. Will remove.

Reply:
Removed Namespace manifest.